src/main/java/thePackmaster/cards/farmerpack/SowTheSeeds.java

```diff
@@ -29,34 +36,47 @@ public SowTheSeeds() {
 
     public void use(AbstractPlayer p, AbstractMonster m) {
         atb(new MakeTempCardInHandAction(new Fertilizer()));
-        Iterator<AbstractCard> typeDetect = AbstractDungeon.player.drawPile.group.iterator();
-        while (typeDetect.hasNext()) {
-            AbstractCard i = typeDetect.next();
-            if (i.type == CardType.ATTACK && !attack) {
-                count += 1;
-                att(new DrawSpecificCardAction(i));
-                attack = true;
-            } else if (i.type == CardType.SKILL && !skill) {
-                count += 1;
-                att(new DrawSpecificCardAction(i));
-                skill = true;
-            } else if (i.type == CardType.POWER && !power) {
-                count += 1;
-                att(new DrawSpecificCardAction(i));
-                power = true;
-            } else if (i.type == CardType.STATUS && !status) {
-                count += 1;
-                att(new DrawSpecificCardAction(i));
-                status = true;
-            } else if (i.type == CardType.CURSE && !curse) {
-                count += 1;
-                att(new DrawSpecificCardAction(i));
-                curse = true;
-            }
-            if (count >= magicNumber) {
-                break;
+        addToBot(new AbstractGameAction() {
+            @Override
+            public void update() {
+                isDone = true;
+
+
+                ArrayList<AbstractCard> targets = new ArrayList<>();
+                for (AbstractCard c : AbstractDungeon.player.drawPile.group) {
+                    if (SpireAnniversary5Mod.cardParentMap.get(c.cardID) != null) {

```

Comment:
You should get rid of this check -- it was what Booster Tutor needed (so only cards in packs were considered), but Sow the Seeds doesn't need that.

Instead, Sow the Seeds can just do `targets.add(c)`, without the `if` surrounding it.